homeassistant/components/linky/__init__.py

```diff
@@ -47,9 +47,15 @@
 
 async def async_setup_entry(hass: HomeAssistantType, entry: ConfigEntry):
     """Set up Linky sensors."""
-
     hass.async_create_task(
         hass.config_entries.async_forward_entry_setup(entry, "sensor")
     )
+    return True
+
 
+async def async_unload_entry(hass: HomeAssistantType, entry: ConfigEntry):
+    """Unload Linky sensors."""
+    hass.async_create_task(
+        hass.config_entries.async_forward_entry_unload(entry, "sensor")
```

Comment:
This should be awaited and not create a task for it.

Reply:
Whoooops, I can fix this during lunch break.

Actually, I took exemple from cert_expiry PR #27150, what makes linky different from this component ?
Or maybe it's just a better practice.

Is it ok for you if I tag you on the next PR ?

Reply:
That PR is also not according to standards.

Reply:
PR created #27857